src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -383,16 +383,35 @@ export default class AircraftController {
     }
 
     /**
+     * Finds an aircraft by its callsign
+     *
      * @method findAircraftByCallsign
-     * @param  {string} [callsign='']
+     * @param  {string} [callsign=null]
+     * @return {AircraftModel|null}
+     */
+    findAircraftByCallsign(callsign = null) {
+        if (!callsign) {
+            return null;
+        }
+
+        const normalizedCallSign = callsign.toUpperCase();
```

Comment:
@n8rzz I tried to evolve your previous comment here (see `AircraftModel get callsign()`)